worker-tasks/sbt

```diff
@@ -0,0 +1,580 @@
+#!/usr/bin/env bash
+#
+# A more capable sbt runner, coincidentally also called sbt.
+# Author: Paul Phillips <paulp@improving.org>
+
```

Comment:
I dislike this file as well, but the pattern we have established over time (by those who use Scala the most) is that this snippet gets added to the repository wherever it needs to be. I'm hesitant to change that without knowing more about the wrapper itself, or `sbt` in general.